components/atom/spinner/src/SUILoader/index.scss

```diff
@@ -0,0 +1,45 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-accent-atom-spinner-loader: color-variation($c-accent, 1) !default;
+$c-primary-atom-spinner-loader: color-variation($c-primary, -1) !default;
+$sz-atom-spinner-oval: $sz-icon-s;
+$sz-atom-spinner: $sz-atom-spinner-oval * 2;
+
+@keyframes sui-atom-spinner {
+  0% {
+    box-shadow: -$sz-atom-spinner-oval $sz-atom-spinner-oval $c-accent-atom-spinner-loader, 0 $sz-atom-spinner $c-primary-atom-spinner-loader;
+  }
+
+  25% {
+    box-shadow: 0 $sz-atom-spinner-oval $c-accent-atom-spinner-loader, -$sz-atom-spinner-oval $sz-atom-spinner $c-primary-atom-spinner-loader;
+  }
+
+  50% {
+    box-shadow: 0 $sz-atom-spinner $c-accent-atom-spinner-loader, -$sz-atom-spinner-oval $sz-atom-spinner-oval $c-primary-atom-spinner-loader;
+  }
+
+  75% {
+    box-shadow: -$sz-atom-spinner-oval $sz-atom-spinner $c-accent-atom-spinner-loader, 0 $sz-atom-spinner-oval $c-primary-atom-spinner-loader;
+  }
+
+  100% {
+    box-shadow: -$sz-atom-spinner-oval $sz-atom-spinner-oval $c-accent-atom-spinner-loader, 0 $sz-atom-spinner $c-primary-atom-spinner-loader;
+  }
+}
+
+$anim-atom-spinner: sui-atom-spinner 1.5s ease-in-out infinite;
+
+.sui-atom-spinner {
+  &-loader {
+    animation: $anim-atom-spinner;
+    background-color: transparent;
+    border-radius: 50%;
+    box-shadow: -$sz-atom-spinner-oval $sz-atom-spinner-oval $c-accent-atom-spinner-loader, 0 $sz-atom-spinner $c-primary-atom-spinner-loader;
+    height: $sz-atom-spinner-oval;
+    left: 50%;
+    margin: 0 auto;
+    position: absolute;
+    top: calc(50% - #{$sz-atom-spinner});
+    width: $sz-atom-spinner-oval;
```

Comment:
IE don´t loves 'calc'

Reply:
We support IE >= 11 and it looks like calc is IE >= 9 https://developer.mozilla.org/es/docs/Web/CSS/calc

Also I've tested it and it seems to be working

Reply:
ok :)